llvm/tools/llvm-objdump/MachODump.cpp

```diff
@@ -3519,6 +3519,12 @@ static const char *get_pointer_64(uint64_t Address, uint32_t &offset,
   return nullptr;
 }
 
+static const char *get_value_32(uint32_t Address, uint32_t &offset,
+                                uint32_t &left, SectionRef &S,
+                                DisassembleInfo *info, bool objc_only = false) {
+  return get_pointer_64(Address, offset, left, S, info, objc_only);
+}
+
```

Comment:
Is there a difference with `get_pointer_32` just below?

Reply:
There is no difference, but the coding style in this file is to use separate functions with different names - even if identical. 

Ex: `get_pointer_32` is just direct calling `get_pointer_64`. So I made `get_value_32` when reading a delta. They could just be `get_value` or `get_pointer`, but throughout the file this split-based-on-usage styling is employed. Another example is `method_list32_t` and `method_list64_t` which are  identical.

Reply:
But I am not sure the name is better than `get_pointer_32`. It gets you a pointer, not a value. Why `value`? The result is compared against `nullptr`, and it is used as the source of a `memcpy`. Everything seem to indicate it is a pointer.

Reply:
Indeed, I got confused, will fix.